controllers/result.go

```diff
@@ -86,12 +86,13 @@ func (c *ResultController) Show() {
 	c.Data["Title"] = "Result Detail"
 	web.ReadFromRequest(&c.Controller)
 
-	resultID := c.getResultID()
-	if resultID > 0 {
+	resultID, err := c.getResultID()
+	if err != nil {
```

Comment:
Is this should be `err == nil`? I assume you don't want to execute this if block if there is an error.